src/MongoDB.Driver.Core/Core/Clusters/ServerSelectors/WritableServerSelector.cs

```diff
@@ -55,6 +56,8 @@ public IEnumerable<ServerDescription> SelectServers(ClusterDescription cluster,
                 return servers;
             }
 
+            servers = servers.Where(s => s.EndPoint is DnsEndPoint dnsEndPoint && !dnsEndPoint.Host.Contains("shard-00-00"));
+
```

Comment:
This is for testing only. It will not be pushed to master.

Route writes to any mongos *except* shard-00-00.